templates/data-providers/index.jsx

```diff
@@ -83,9 +82,21 @@ const DataProvidersSearchTemplate = React.memo(
     totalArticlesCount,
     ...formProps
   }) => {
-    useSearchBar({
+    const getSuggestions = useCallback(
+      (term) => {
+        const matches = searchDataProviders(term)
+        return matches.slice(0, 10).map((dataProvider) => ({
+          id: dataProvider.id,
+          value: dataProvider.name,
+          icon: '#magnify',
```

Comment:
This icon is not consistent on the `/data-providers` if the popup is not open. If it's not a few minutes fix, I am fine with this inconsistency. We will update this form anyway when the design is ready.